patches/components-search_engines-search_engine_type.h.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/components/search_engines/search_engine_type.h b/components/search_engines/search_engine_type.h
+index e1228cbf36a79367d10319d6c8f8661cc2978916..0102983617da898c563240bfef3df8fc39bf3f1f 100644
+--- a/components/search_engines/search_engine_type.h
++++ b/components/search_engines/search_engine_type.h
+@@ -98,6 +98,7 @@ enum SearchEngineType {
+   SEARCH_ENGINE_FREESPOKE = 78,
+   SEARCH_ENGINE_KAGI = 79,
+   SEARCH_ENGINE_STARTER_PACK_AI_MODE = 80,
++  SEARCH_ENGINE_STARTER_PACK_ASK_BRAVE_SEARCH = 81,
```

Comment:
`#define SEARCH_ENGINE_MAX ...` similar to https://github.com/brave/brave-core/pull/31149/files#r2350205843